subsys/ipc/ipc_service/backends/ipc_rpmsg_static_vrings_mi.c

```diff
@@ -100,7 +100,7 @@ static struct rpmsg_mi_instance *get_available_instance(const struct ipc_ept_cfg
 static struct ipc_rpmsg_ept *get_available_ept_slot(struct ipc_rpmsg_instance *rpmsg_instance)
 {
 	for (size_t i = 0; i < NUM_ENDPOINTS; i++) {
-		if (rpmsg_instance->endpoint[i].name == NULL) {
+		if (strcmp(rpmsg_instance->endpoint[i].name, "") == 0) {
```

Comment:
nit: simple comparison would be more efficient
```suggestion
		if (rpmsg_instance->endpoint[i].name[0] == '\0') {
```

Reply:
oh, this is nice :+1: